### DISCUSSION TITLE
Arena spectator NPC

### DISCUSSION THREAD

**Discussion by unitscan** [chromiecraft/chromiecraft#6022] [Suggestions]
If my memory serves me right, I remember an Arena spectator NPC on other AzerothCore servers. Have you already thought about this idea?

**Comment by r-o-b-o-t-o** [comment]
You can use the command `.spect spectate Playername` to see a player's arena match

**Comment by Helias** [comment]
After the release of the NPC spectator where any player can easily spectate matches without any commands OR check the NPC info description for the commands, I think we can close this github discussion.

Thanks for the discussion.

References:
[youtube.com/watch?v=P7hYa2rwWqo](https://www.youtube.com/watch?v=P7hYa2rwWqo)
[chromiecraft.com/en/pvp-revamp-arena-s3](https://www.chromiecraft.com/en/pvp-revamp-arena-s3/)
